charts/akeyless-api-gateway/templates/cache.yaml

```diff
@@ -69,9 +69,15 @@ spec:
                 fi
           initialDelaySeconds: 5
           periodSeconds: 5
+        resources:
+{{- toYaml .Values.cache.resources | nindent 12 }}
         env:
         - name: REDIS_DISABLE_COMMANDS
           value: "FLUSHDB,FLUSHALL"
+        - name: REDIS_MAXMEMORY_POLICY
+          value: "allkeys-lru"
+        - name: REDIS_MAXMEMORY
```

Comment:
remove redisMaxmemory 
REDIS_MAXMEMORY param should be set by resources.limits.memory

Reply:
So the cache.resources.limits.memory must be set.

Reply:
Redis itself requires some memory overhead for internal operations, so setting `REDIS_MAXMEMORY` to the exact value of the pod’s memory limit might cause Redis to hit the limit and be terminated. 
**To avoid this,** we might want to set `REDIS_MAXMEMORY` to a **slightly lower** value than the pod’s memory limit.
Therefor i think will should use separat param for it.

Reply:
A safe approach might be to set `REDIS_MAXMEMORY` to around 90-95% of the pod’s resources.limits.memory to leave some buffer for Redis overhead and any other processes.

Reply:
Done